canopy/app/saplings/svelte.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * Copyright 2019 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+!(function() {
```

Comment:
I am not sure about committing the transpiled code here. What does the experience look like while we are developing the initial saplings? Where would the non-transpiled code live?

Reply:
Good point. I think adding just the vanilla one here as an example would be a good compromise.

The other Saplings can live in `splinter/canopy/saplings` we can then add a pre-install script to build and move them.

Reply:
I think that would be a good approach